content/packages/fs-mirrors.txt

```diff
@@ -0,0 +1 @@
+http://openra.ppmsite.com/fs-base.zip
```

Comment:
For the sake of a consistent naming scheme please call this `ts-fs-base.zip` or `ts-expand-base.zip`.  This keeps each mod's packages grouped logically on the servers.